build-zip.sh

```diff
@@ -113,10 +109,17 @@ function generate_zip() {
 #_______________________________________________________________________________
 #                                 Main
 echo "~~~ Downloading repo indexes"
-apps_config | awk '{print $1}' | uniq | xargs -l bash -c 'download_repo_index $@' -
+download_repo_index fdroid
+download_repo_index microg
```

Comment:
This is an unwanted regression because this doesn't support adding new repos with a single line in the `get_repo_base_url()` function. So don't rewrite this, thanks.